sandbox/web_sandbox/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(web_sandbox VERSION 0.1 LANGUAGES CXX)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(MU_ROOT ${CMAKE_CURRENT_LIST_DIR}/../..)
+set(PROJECT_ROOT_DIR ${MU_ROOT})
+
+set(CMAKE_MODULE_PATH
+    ${MU_ROOT}/buildscripts/cmake
+    ${CMAKE_MODULE_PATH}
+)
+
+include(SetupBuildEnvironment)
+include(GetPlatformInfo)
+include(DeclareModuleSetup)
+
+# Qt
+set(qt_components Core Gui Quick Qml Widgets)
+set(qt_libs Qt6::Core Qt6::Gui Qt6::Quick Qt6::Qml Qt6::Widgets)
+set(qt_includes )
+if (OS_IS_LIN)
+    list(APPEND qt_components DBus)
+    list(APPEND qt_libs Qt::DBus)
+endif()
+
+find_package(Qt6 6.2 REQUIRED COMPONENTS ${qt_components})
+
+foreach(qlib ${qt_libs} )
+    get_target_property(Qt6X_includes ${qlib} INTERFACE_INCLUDE_DIRECTORIES)
+    if (Qt6X_includes STREQUAL "Qt6X_includes-NOTFOUND")
+
+    else()
+        list(APPEND qt_includes ${Qt6X_includes})
+    endif()
+endforeach()
+
+if (OS_IS_WASM)
+    qt_add_library(all_qml_plugins STATIC)
+    qt_import_qml_plugins(all_qml_plugins)
```

Comment:
This only scans folders that are subfolders of the folder where the `all_qml_plugins` target was created (so it won't scan files outside sandbox/web_sandbox)